tests/test_series.py

```diff
@@ -832,3 +832,116 @@ def test_categorical_codes():
     # GH-111
     cat = pd.Categorical(["a", "b", "a"])
     assert_type(cat.codes, "np_ndarray_int")
+
+
+def test_string_accessors():
+    s = pd.Series(
+        ["applep", "bananap", "Cherryp", "DATEp", "eGGpLANTp", "123p", "23.45p"]
+    )
+    s2 = pd.Series([["apple", "banana"], ["cherry", "date"], [1, "eggplant"]])
+    s3 = pd.Series(["a1", "b2", "c3"])
+    check(assert_type(s.str.capitalize(), pd.Series), pd.Series)
+    check(assert_type(s.str.casefold(), pd.Series), pd.Series)
+    check(assert_type(s.str.cat(sep="X"), str), str)
+    check(assert_type(s.str.center(10), pd.Series), pd.Series)
+    check(assert_type(s.str.contains("a"), "pd.Series[bool]"), pd.Series, bool)
+    check(assert_type(s.str.count("pp"), "pd.Series[int]"), pd.Series, int)
+    check(assert_type(s.str.decode("utf-8"), pd.Series), pd.Series)
+    check(assert_type(s.str.encode("latin-1"), pd.Series), pd.Series)
+    check(assert_type(s.str.endswith("e"), "pd.Series[bool]"), pd.Series, bool)
+    check(assert_type(s3.str.extract(r"([ab])?(\d)"), pd.DataFrame), pd.DataFrame)
+    check(assert_type(s3.str.extractall(r"([ab])?(\d)"), pd.DataFrame), pd.DataFrame)
+    check(assert_type(s.str.find("p"), pd.Series), pd.Series)
+    # Commented out since not easy to test Series[list]
+    # check(assert_type(s.str.findall("pp"), pd.Series), pd.Series)
```

Comment:
See above - have to change return time of `findall()` to allow test to work

Reply:
Fixed.